app/controllers/state_file/state_file_pages_controller.rb

```diff
@@ -8,6 +8,8 @@ def redirect_locale_home
     end
 
     def fake_direct_file_transfer_page
+      # TODO: make sure we can't reach this page on prod
```

Comment:
Should we add an environment check? What needs to be done here?

Reply:
@mpidcock yeah actually we did changes here instead so going to remove all the link stuff from this PR https://www.pivotaltracker.com/story/show/186917213